.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Official Discord server
+  - name: Discord Support
     url: https://discord.premid.app
-    about: Please use our ticket system for urgent requests!
+    about: If you are experiencing issues with your PreMiD, go to our Discord server for help.
```

Comment:
Should still mention the ticket system, unless you want people to just message in a random chat which is also fine